packages/experience/package.json

```diff
@@ -73,7 +73,7 @@
     "react-device-detect": "^2.2.3",
     "react-dom": "^18.3.1",
     "react-helmet": "^6.1.0",
-    "react-hook-form": "^7.34.0",
+    "react-hook-form": "^7.53.0",
```

Comment:
can you upgrade it from all pacakges?

Reply:
pre offline discussion, upgrade console package in another PR, since code adjustment required.